src/hooks/scroll.js

```diff
@@ -1,9 +1,10 @@
 import { useState, useEffect } from 'react'
+import { IS_SERVER } from '../constants'
 
 export function useScroll() {
   const [pos, setPos] = useState({
-    top: window.scrollY,
-    left: window.scrollX
+    top: IS_SERVER ? 0 : window.scrollY,
```

Comment:
I noticed that you didn't check for `window` in the `useEffect` callback. Is that because `window` is defined by the time it is called? If so, could we avoid this check by always defaulting to `0` here (or null) and then updating the state to the true values in `useEffect`? 

The media-controls hook had to use that approach to set initial values as soon as the `ref` is [rendered](https://github.com/nearform/react-browser-hooks/blob/master/src/hooks/media-controls.js#L56:L59) but [not before](https://github.com/nearform/react-browser-hooks/blob/master/src/hooks/media-controls.js#L4:L8). It works well and I noticed that it didn't need changes for SSR, so maybe that's a pattern we can adopt to support SSR without needing these specific checks?

Reply:
`useEffect` doesn't run on the server, just when it mounts in the DOM (same as `componentDidMount`), so we're safe to access `window` there.

The pattern of using static defaults and then updating state with `window` values within `useEffect` will work but it will introduce an extra render as the state will always update once it's mounted. What do you think?

Reply:
That's a good point. It's probably more important to avoid unnecessary renders. I'll merge this as it is

Reply:
Any chance you can bump package.json?